From c46f6cb84fb189da723a9932a5195a553762e2b6 Mon Sep 17 00:00:00 2001
Message-Id: <c46f6cb84fb189da723a9932a5195a553762e2b6.1684168831.git.stefan@agner.ch>
From: Stefan Agner <stefan@agner.ch>
Date: Tue, 22 Nov 2022 10:51:17 +0100
Subject: [PATCH] Support custom platform tag

---
 src/controller/python/BUILD.gn | 20 ++++++++++----------
 1 file changed, 10 insertions(+), 10 deletions(-)

diff --git a/src/controller/python/BUILD.gn b/src/controller/python/BUILD.gn
index 750de2b837..424ed433b1 100644
--- a/src/controller/python/BUILD.gn
+++ b/src/controller/python/BUILD.gn
@@ -33,6 +33,15 @@ config("controller_wno_deprecate") {
 declare_args() {
   chip_python_version = "0.0"
   chip_python_package_prefix = "chip"
+
+  if (current_os == "mac") {
+    chip_python_platform_tag = string_replace(
+            string_replace(mac_deployment_target, "macos", "macosx."),
+            ".",
+            "_")
+  } else {
+    chip_python_platform_tag = current_os
+  }
 }
 
 shared_library("ChipDeviceCtrl") {
@@ -326,16 +335,7 @@ chip_python_wheel_action("chip-core") {
     cpu_tag = current_cpu
   }
 
-  if (current_os == "mac") {
-    py_platform_tag = string_replace(
-            string_replace(mac_deployment_target, "macos", "macosx."),
-            ".",
-            "_")
-  } else {
-    py_platform_tag = current_os
-  }
-
-  py_platform_tag = py_platform_tag + "_" + cpu_tag
+  py_platform_tag = chip_python_platform_tag  + "_" + cpu_tag
 
   tags = "cp37-abi3-" + py_platform_tag
 
-- 
2.40.1

