From 44cc7593f34d07501c67f06e5f6d7fcf719aae71 Mon Sep 17 00:00:00 2001
From: Stefan Agner <stefan@agner.ch>
Date: Tue, 22 Nov 2022 10:51:17 +0100
Subject: [PATCH] Support custom platform tag

---
 src/controller/python/BUILD.gn | 20 ++++++++++----------
 1 file changed, 10 insertions(+), 10 deletions(-)

diff --git a/src/controller/python/BUILD.gn b/src/controller/python/BUILD.gn
index 5fc2212098..caa33c3a40 100644
--- a/src/controller/python/BUILD.gn
+++ b/src/controller/python/BUILD.gn
@@ -35,6 +35,15 @@ declare_args() {
   chip_python_version = "0.0"
   chip_python_package_prefix = "chip"
   chip_python_supports_stack_locking = chip_system_config_locking != "none"
+
+  if (current_os == "mac") {
+    chip_python_platform_tag = string_replace(
+            string_replace(mac_deployment_target, "macos", "macosx."),
+            ".",
+            "_")
+  } else {
+    chip_python_platform_tag = current_os
+  }
 }
 
 shared_library("ChipDeviceCtrl") {
@@ -344,16 +353,7 @@ chip_python_wheel_action("chip-core") {
     cpu_tag = current_cpu
   }
 
-  if (current_os == "mac") {
-    py_platform_tag = string_replace(
-            string_replace(mac_deployment_target, "macos", "macosx."),
-            ".",
-            "_")
-  } else {
-    py_platform_tag = current_os
-  }
-
-  py_platform_tag = py_platform_tag + "_" + cpu_tag
+  py_platform_tag = chip_python_platform_tag  + "_" + cpu_tag
 
   tags = "cp37-abi3-" + py_platform_tag
 
-- 
2.45.2

