From f26db733ebfed0e33f7a0df6fc468e820ddc94be Mon Sep 17 00:00:00 2001
From: Stefan Agner <stefan@agner.ch>
Date: Fri, 27 May 2022 16:38:14 +0200
Subject: [PATCH] Use /data as platform storage location

---
 src/platform/Linux/BUILD.gn | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/src/platform/Linux/BUILD.gn b/src/platform/Linux/BUILD.gn
index 35b5047c9f..4886bc3ebb 100644
--- a/src/platform/Linux/BUILD.gn
+++ b/src/platform/Linux/BUILD.gn
@@ -38,6 +38,12 @@ if (chip_mdns == "platform") {
 }
 
 static_library("Linux") {
+  defines = [
+    "FATCONFDIR=\"/data\"",
+    "SYSCONFDIR=\"/data\"",
+    "LOCALSTATEDIR=\"/data\"",
+  ]
+
   sources = [
     "../DeviceSafeQueue.cpp",
     "../DeviceSafeQueue.h",
-- 
2.45.2

