From 61ab506f9c0cefde811532415b2cc5fe1b84b84f Mon Sep 17 00:00:00 2001
From: Stefan Agner <stefan@agner.ch>
Date: Fri, 27 May 2022 16:38:14 +0200
Subject: [PATCH] Use /data as platform storage location

---
 src/platform/Linux/BUILD.gn | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/src/platform/Linux/BUILD.gn b/src/platform/Linux/BUILD.gn
index 7961b3d648..7ec0d6472d 100644
--- a/src/platform/Linux/BUILD.gn
+++ b/src/platform/Linux/BUILD.gn
@@ -34,6 +34,12 @@ if (chip_mdns == "platform") {
 }
 
 static_library("Linux") {
+  defines = [
+    "FATCONFDIR=\"/data\"",
+    "SYSCONFDIR=\"/data\"",
+    "LOCALSTATEDIR=\"/data\"",
+  ]
+
   sources = [
     "../DeviceSafeQueue.cpp",
     "../DeviceSafeQueue.h",
-- 
2.47.0

